docs/production/native_cloud.md

```diff
@@ -0,0 +1,29 @@
+# Mage and Native Cloud Environments
+Just like a traditional notebook, Mage supports execution in native cloud environments. We provide a set of utility scripts that allow you quickly get started running Mage in the your private cloud. The following services currently have launch scripts
+- [Amazon EC2](#ec2)
+
+Below are guides on how to use these launch scripts
+## Amazon EC2
+We provide the `scripts/run_ec2.sh` script to launch Mage in an EC2 instance. Currently only connections via SSH are supported.
+
+### Prerequistes
+The only prerequisite is that your EC2 instance must have `python3` installed.
+
+### Usage
+To run Mage in an EC2 instance, you need to provide
+- Path to the key pair used with the EC2 instance
+- Your EC2 instance username
+- Your EC2 public DNS name
+
+```bash
+./scripts/run_ec2.sh path_to_key_pair ec2_user_name ec2_public_dns_name [--name custom_repo_name]
+```
+This script will
+1. Connect to your EC2 instance
+2. Install Mage
```

Comment:
Do we need to install the library every time we run the server?

Reply:
I thought to install the library each time since a user might be working with a newly initialized instance of EC2 which may not have Mage installed. The alternative is to make having Mage installed on the EC2 instance a prerequisite for running this script or perform some checking with `pip list | grep mage-ai` to see if Mage is already installed.

Reply:
Let's check whether the mage-ai library is installed.

Reply:
Added check so mage is only installed if not already installed